guide/14-deep-learning/yolov3_object_detector.ipynb

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
<p>.. concatenation of feature layers with earlier feature layers which <strong>preserve</strong> fine-grained features. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>